test/unittest/statistics/dds/StatisticsDomainParticipantTests.cpp

```diff
@@ -136,11 +136,7 @@ class StatisticsDomainParticipantTests : public ::testing::Test
     void helper_block_for_at_least_entries(
             uint32_t amount)
     {
-        std::unique_lock<std::mutex> lck(*mutex_);
```

Comment:
I think this mutex is only used here. Remove it.

Reply:
done